NIST_ruby/single_GE/include/ruby_config.yml

```diff
@@ -1,4 +1,4 @@
-analysis_name: results_ruby_hexrd08_py38_scan_0  # defaults to analysis
+analysis_name: ruby/scan-0  # defaults to analysis
```

Comment:
I like these shorter like this! :slightly_smiling_face:

Reply:
Good point on checking the tests. I'll do that.

I like the shorter names too, but there is a possible issue there.  It works fine with fit-grains because hexrd only appends things to the analysis name. But it won't work with find-orientations CLI (but is OK running find-orientations from fit-grains) because it prepends "accepted-orientations" to the analysis name.  

I was thinking it might nice to have a separate, simple pull request updating the names based on the analysis_name.  It's a little strange that it makes a directory with that name and puts fit-grains outputs there, but fnd-orientations creates files at the same level with that name.  I am thinking we should just make the directory and put `accepted-orientations.txt`  inside it, instead of naming it `accepted-orientatons-<analysis-name>.dat`. Also, if you put a "/" in the analysis_name (as above), that directory has to exist. We could fix that too.

Reply:
Those things sound great to me.